proto/vnet_mapping.proto

```diff
@@ -7,7 +7,7 @@ import "route_type.proto";
 
 message VnetMapping {
     // reference to routing type
-    route_type.RoutingType action_type = 1;
+    route_type.RoutingType routing_type = 1;
```

Comment:
I feel we should also add some PRs in sonic-swss and sonic-mgmt for this renaming.

Reply:
I will include the changes with my PL implementation PR for swss